src/DacpacTool/BuildOptions.cs

```diff
@@ -12,6 +12,7 @@ public class BuildOptions : BaseOptions
         public FileInfo InputFile { get; set; }
         public string[] Reference { get; set; }
         public string[] Property { get; set; }
+        public string[] CreateScriptProperty { get; set; }
```

Comment:
I'm a bit on the fence about the name of this property. Technically they are used when generating the create script, but essentially they are deployment options, so perhaps `DeployProperty` would be better name? And then rename `Property` to `BuildProperty` as well, to make the distinction clear?

Reply:
I've renamed the properties as suggested